### PR TITLE
update to 11.3.0

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -220,14 +223,14 @@ outputs:
...
 
-        - test -f "${PREFIX}/lib/libgcc_s.1.1.dylib"        # [target_platform == "osx-arm64" or gfortran_version == "11.2.0"]
+        - test -f "${PREFIX}/lib/libgcc_s.1.1.dylib"        # [target_platform == "osx-arm64" or gfortran_version == "11.3.0"]
 
 about:
-  home: http://gcc.gnu.org/
+  home: https://gcc.gnu.org/
   license: GPL-3.0-only WITH GCC-exception-3.1
   license_family: GPL
   license_file: COPYING3
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:236]: Missing:
- dev_url: https://gcc.gnu.org/git/?p=gcc.git;a=tree
- doc_url: https://gcc.gnu.org/onlinedocs/

File: recipe/meta.yaml [hunk 2]
```diff
@@ -220,14 +223,14 @@ outputs:
...
+        - test -f "${PREFIX}/lib/libgcc_s_ppc64.1.dylib"    # [target_platform == "osx-64" and gfortran_version != "11.3.0"]
+        - test -f "${PREFIX}/lib/libgcc_s_x86_64.1.dylib"   # [target_platform == "osx-64" and gfortran_version != "11.3.0"]
 
-        - test -f "${PREFIX}/lib/libgcc_s.1.1.dylib"        # [target_platform == "osx-arm64" or gfortran_version == "11.2.0"]
+        - test -f "${PREFIX}/lib/libgcc_s.1.1.dylib"        # [target_platform == "osx-arm64" or gfortran_version == "11.3.0"]
 
 about:
-  home: http://gcc.gnu.org/
+  home: https://gcc.gnu.org/
   license: GPL-3.0-only WITH GCC-exception-3.1
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:234]: I think the license should be `GPL-3.0-or-later`, see https://gcc.gnu.org/onlinedocs/gcc-11.3.0/gfortran/Copying.html#Copying. 
I didn't find GCC Runtime Library Exception, see the text https://www.gnu.org/licenses/gcc-exception-3.1.html

Comment (skupr-anaconda) [reply recipe/meta.yaml:234]: `GPL-3.0-with-GCC-exception` is deprecated https://spdx.org/licenses/GPL-3.0-with-GCC-exception.html

Comment (katietz) [reply recipe/meta.yaml:234]: no, it is with exception-3.1 that is important ... it is not just GPL 3.0-or-later

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml:236]: thanks, done

**Review by skupr-anaconda — APPROVED** [review]

**Review by cbouss — APPROVED** [review]
LGTM

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**